dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/domain/event/LogEventMapperWrapper.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.log.internal.domain.event
+
+import com.datadog.android.core.internal.utils.devLogger
+import com.datadog.android.event.EventMapper
+import com.datadog.android.log.model.LogEvent
+import java.util.Locale
+
+internal class LogEventMapperWrapper(private val wrappedEventMapper: EventMapper<LogEvent>) :
```

Comment:
Maybe we could have a generic `ConsistentMapper` that perform this (with an option to allow dropping an event or not, and use that for both logs and spans ?

Reply:
I wanted to actually do this but then I realized that each feature has its own special case :
- RUM treats the ViewEvent differently than the others
- TRACE doesn't accept null from the EventMapper
- LOGS accepts null

Having such a class conditioned by a strategy will only add too much complexity and later if we want to change something in a feature it will be a lot easier to just modify one of these strategies.